atom/browser/web_contents_preferences.cc

```diff
@@ -216,6 +217,25 @@ bool WebContentsPreferences::IsSandboxed(content::WebContents* web_contents) {
 }
 
 // static
+bool WebContentsPreferences::ConvertValueToIntegerFromString(
+    WebContentsPreferences* pref, std::string attributeName, int* intValue) {
+
+  // if it is already an integer, no conversion needed
+  if (pref->web_preferences_.GetInteger(attributeName, intValue)) {
+    return true;
+  }
+
+  std::string stringValue;
+
+  if (pref->web_preferences_.GetString(attributeName, &stringValue)) {
+    std::stringstream(stringValue) >> *intValue;
```

Comment:
Noted! Thanks for pointing it out, I am still relatively new to the codebase. :)